src/util/validateActionRegistrations.ts

```diff
@@ -0,0 +1,22 @@
+import { actionMetadataRegistry } from '../@types/ActionMetadata'
+import ActionType from '../@types/ActionType'
+import * as actions from '../actions'
+
+/**
+ * Validates that all actions have been registered in the metadata registry.
+ */
+export default function validateActionRegistrations(): void {
```

Comment:
Let's move this into the `util` module so that `actionMetadataRegistry` can be encapsulated.